templates/admin/mentoring/index.html.twig

```diff
@@ -19,18 +17,20 @@
         </thead>
         <tbody>
         {% for mentoring in mentorings %}
-        <tr>
-            <th scope="row">{{ mentoring.id }}</th>
-            <td>{{ mentoring.student.user.firstname }}</td>
-            <td>{{ mentoring.student.user.lastname }}</td>
-            <td>{{ mentoring.mentor.user.firstname }}</td>
-            <td>{{ mentoring.mentor.user.lastname }}</td>
-            {% if mentoring.isAccepted == true %}
-                <td><a class="btn btn-danger" href="{{ path('admin_mentoring_desactivate', {'id': mentoring.id}) }}) }}">Désactiver le mentorat</a></td>
-            {% else %}
-            <td><a class="btn btn-warning" href="#">Le mentorat est désactivé</a></td>
-            {% endif %}
-        </tr>
+        {% if mentoring.student != null %}
```

Comment:
Hello there, I'm a bit surprised to see a " != " in twig . It should be "is not null" as in the doc= https://twig.symfony.com/doc/2.x/tests/null.html + it could be a good idea to test the mentoring.mentor as well